tests/test_filters.py

```diff
@@ -234,10 +234,11 @@ def test_wireframe_composite():
 
 def test_contour():
     dataset = examples.load_uniform()
-    iso = dataset.contour()
-    assert iso is not None
-    iso = dataset.contour(isosurfaces=[100, 300, 500])
-    assert iso is not None
+    for method in [None, 'contour', 'marching_cubes', 'flying_edges']:
+        iso = dataset.contour(method=method)
+        assert iso is not None
+        iso = dataset.contour(isosurfaces=[100, 300, 500], method=method)
+        assert iso is not None
```

Comment:
A better way to do this would be to paramaterize the test itself so that way if one of the algorithms fails, we know which one. You can do this by:

```py
@pytest.mark.parametrize('method', [None, 'contour', 'marching_cubes', 'flying_edges'])
def test_contour(method):
    dataset = examples.load_uniform()
    iso = dataset.contour(method=method)
    assert iso is not None
    iso = dataset.contour(isosurfaces=[100, 300, 500], method=method)
    assert iso is not None
    ...
```

Reply:
Ah nice! Learning new python practices/tricks along the way was something I was also hoping for while contributing here :)